contracts/upgradeable-Bridge/FiberRouter.sol

```diff
@@ -16,6 +17,7 @@ contract FiberRouter is Ownable, TokenReceivable {
     using SafeERC20 for IERC20;
     address public pool;
     address public oneInchAggregatorRouter;
+    address public constant WETH = 0xc778417E063141139Fce010982780140Aa0cD5Ab; //need to change for every blockchain
```

Comment:
Don't use this. See comments

Reply:
Thanks @naiemk 

@SibghatUllah1997 please see the approach above. Share if you have any questions.